pyproject.toml

```diff
@@ -51,7 +51,7 @@ bootstrap_config_reader = "databricksbundle.bootstrap.package_config_reader:read
 [pyfony.bootstrap]
 container_init = "pyfonycore.container.container_init:init_with_current_bundle"
 root_module_name = "databricksbundle"
-allowed_environments = ["test_aws", "test_azure"]
+allowed_environments = ["test_aws", "test_azure", "test_daipe_help"]

```

Comment:
this probably isn't necessary, we don't need a special config for testing, we can use `config.yaml`